app/models/assignment_repo/creator.rb

```diff
@@ -135,6 +135,7 @@ def push_starter_code!(github_repo_id)
       starter_code_repository = GitHubRepository.new(client, starter_code_repo_id)
 
       assignment_repository.get_starter_code_from(starter_code_repository)
+      setup_repository(starter_code_repository, assignment_repository)
```

Comment:
Two comments:
I haven't read past here in the diff, but `setup_repository` doesn't really say much about what exactly this method does.

Also I'm not sure this belongs in `push_starter_code`